p11mod/p11mod.go

```diff
@@ -410,9 +411,34 @@ func (ll *llBackend) CreateObject(sh pkcs11.SessionHandle, temp []*pkcs11.Attrib
 }
 
 func (ll *llBackend) CopyObject(sh pkcs11.SessionHandle, o pkcs11.ObjectHandle, temp []*pkcs11.Attribute) (pkcs11.ObjectHandle, error) {
-	// TODO
-	log.Println("p11mod CopyObject: not implemented")
-	return 0, pkcs11.Error(pkcs11.CKR_FUNCTION_NOT_SUPPORTED)
+	session, err := ll.getSessionByHandle(sh)
+	if err != nil {
+		return 0, err
+	}
+
+	// Handles are 1-indexed, while our slice is 0-indexed.
+	objectIndex := int(o - 1)
+
+	if objectIndex < 0 || objectIndex >= len(session.objects) {
+		log.Printf("p11mod CopyObject: index invalid: requested %d, object count %d\n", objectIndex, len(session.objects))
+		return 0, pkcs11.Error(pkcs11.CKR_FUNCTION_NOT_SUPPORTED)
```

Comment:
Putting a blank line between the above `log.Printf` and this line should fix the linter error.

Reply:
This doesn't fix the error, the test passed then failed on the second try.

Reply:
Let me Re-run it

Reply:
Go Lint new test passed